pkg/crypto/keys/publickey.go

```diff
@@ -406,3 +406,9 @@ func (p *PublicKey) UnmarshalYAML(unmarshal func(any) error) error {
 	}
 	return p.DecodeBytes(b)
 }
+
+// CompressedHexString returns the hex string representation of the public key
+// in its compressed form.
+func (p *PublicKey) CompressedHexString() string {
```

Comment:
I think it's time to adjust the comment of `NewPublicKeyFromString`. To something like that, we need to specify that it accepts only compressed representation:
```
NewPublicKeyFromString returns a public key created from the given hex string public key representation in compressed form. 
```

Reply:
May be done in a separate commit.